bitbucket-pipelines.yml

```diff
@@ -29,7 +57,14 @@ definitions:
 pipelines:
   pull-requests:
     '**':
-      - step: *test
+      - step: *install
+      - parallel:
+          steps:
+            - step: *test
+            - step: *lint
+            - step:
+                <<: *deploy-temp-stack
+                deployment: Staging
```

Comment:
We should setup a separate deployment environment just for this e.g. "pr-env".

Also note that if one PR is in progress any new ones will be started in a paused state and need to manually be started once the other completes.

Reply:
Using the default env vars would probably be better so you don't get those collisions.